### DISCUSSION TITLE
Creating tables from typescript types

### DISCUSSION THREAD

**Discussion by rodjoseph** [supabase/supabase#868] [Questions] [answered]
Is there a way for me to create tables in the database from Typescript types, or should I be creating tables manually?

**Comment by silentworks** [comment, marked as answer]
You can create tables with any tool that can connect to the database directly. In the case of using TypeScript types, I don't think these are an actual concrete representation of an object, however, you could use TypeORM and its annotation system to create your models and schema in TypeScript.

**Reply by rodjoseph** [reply]
Excellent, thank you very much for your answer.
